static/instruction.sh

```diff
@@ -3,7 +3,7 @@
 clear
 cat << INST1
 +-----------------------------------------------------------------------+
-| Thank you for downloading this Nextcloud VM made by Tech and Me!      |
+| Thank you for downloading this Nextcloud VM made by the Nextcloud Community!      |
```

Comment:
Have you run the scripts so that you cheked that the text won't be off and symetric to the box?